Home/home.html

```diff
@@ -4,6 +4,219 @@
+  transform: translateY(-50%);
+  cursor: pointer;
+  -webkit-transition: all 0.3s;
+  transition: all 0.3s;
+}
+
+footer.main-footer .newsletter button:hover {
+  color: #3494E6;
+}
+
+footer.main-footer .copyrights {
+  background: #0e0e0e;
+  color: #888;
+  padding: 30px 0;
+}
+
+footer.main-footer .copyrights p {
+  font-weight: 300;
+  font-size: 0.95em;
+}
+
+footer.main-footer .copyrights a {
+  color: inherit;
+}
+
+footer.main-footer .copyrights a:hover {
+  color: #3494E6;
+}
+
+footer.main-footer .copyrights p,
+footer.main-footer .copyrights ul {
+  margin: 0;
+}
+
+footer.main-footer .copyrights ul.payment-list img {
+  width: 35px;
+}
+
+
+</style>
+</head>
+<body>
+    <!-- Footer-->
+    <footer class="main-footer">
+      <!-- Main Block -->
+      <div class="main-block">
+        <div class="container">
+          <div class="row">
+            <div class="info col-lg-4">
+              <div class="logo"><h2>LAEDEX</h2></div>
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing.</p>
+              <ul class="social-menu list-inline">
+                <li class="list-inline-item"><a href="#" target="_blank" title="twitter"><i class="fab fa-twitter"></i></a></li>
+                <li class="list-inline-item"><a href="#" target="_blank" title="facebook"><i class="fab fa-facebook"></i></a></li>
+                <li class="list-inline-item"><a href="#" target="_blank" title="instagram"><i class="fab fa-instagram"></i></a></li>
+                <li class="list-inline-item"><a href="#" target="_blank" title="pinterest"><i class="fab fa-pinterest"></i></a></li>
+                <li class="list-inline-item"><a href="#" target="_blank" title="vimeo"><i class="fab fa-vimeo"></i></a></li>
+              </ul>
+            </div>
+            <div class="site-links col-lg-2 col-md-6">
+              <h5 class="text-uppercase">Title</h5>
+              <ul class="list-unstyled">
+                <li> <a href="#">Link</a></li>
+                <li> <a href="#">Link</a></li>
+                <li> <a href="#">Link</a></li>
+                <li> <a href="#">Link</a></li>
+              </ul>
+            </div>
+            <div class="site-links col-lg-2 col-md-6">
+              <h5 class="text-uppercase">Title</h5>
+              <ul class="list-unstyled">
+                <li> <a href="#">About</a></li>
+                <li> <a href="#">Contact</a></li>
+              </ul>
+            </div>
+            <div class="newsletter col-lg-4">
+              <h5 class="text-uppercase">Title</h5>
+              <p> Lorem ipsum dolor sit amet, consectetur adipisicing elit. At itaque temporibus.</p>
+              <form action="#" id="newsletter-form">
+                <div class="form-group">
+                  <input type="email" name="subscribermail" placeholder="Your Email Address">
+                  <button type="submit"> <i class="fa fa-paper-plane"></i></button>
+                  <p>blabla</p>
```

Comment:
Hello team , well done on working on creating the nav and footer section , however I can see that "blabla" text has no meaning it is better to always use significant terms , but overall good work 👍